src/pages/Products.js

```diff
@@ -178,18 +199,35 @@ const Products = () => {
 							</h2>
 						</div>
 					)}
-					<div className="grid md:grid-cols-2 xl:grid-cols-4 gap-4 w-full p-0 justify-evenly xl:justify-around">
-						{currentProducts?.length ? (
-							currentProducts.map((product) => (
-								<ProductItem data={product} key={product.id} />
-							))
-						) : (
-							<h1>No hay productos usando estos argumentos</h1>
-						)}
-					</div>
+					{
+						currentProducts.length === 0
+						? <h1 className="text-center">No hay productos usando estos argumentos</h1>
+						:
+						<>
+							<div className="grid md:grid-cols-2 xl:grid-cols-4 gap-4 w-full p-0 justify-evenly xl:justify-around">
+								{
+									currentPosts.map((product) => (
+										<ProductItem data={product} key={product.id} />
+									))
+								}
+							</div>
+
+							{currentProducts.length <= 10
+								? null
+								:
+								<Pagination
```

Comment:
Since doesn't exist a condition to "else" of the tertiary operator, u can do the condition without using tertiary operator, to avoid the extra line `? null`

You can do something like this:

```suggestion
							{currentProducts.length <= 10 ||
								<Pagination
```

or

```suggestion
							{currentProducts.length >= 10 &&
								<Pagination
```

To know more about the login applied here u can check this article: https://medium.com/programming-essentials/how-to-conditionally-invoke-a-method-b8817a592712#c3d7